### PR TITLE
Rename CATS Instance Selector ID to CATS Service Contact Instance ID

### PR DESCRIPTION
**Pull request by VMatrix1900**
CIS-ID to CSCI-ID too.

### CODE REVIEW

**Comment by boucadair** [comment]
The change to "CATS Service Contact Instance ID" looks reasonable to me.

File: draft-ietf-cats-framework.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ contributor:
   email: luigi.iannone@huawei.com
  -
   name: Hang Shi
-  org: Huawei Technologies
+  org: Huawei Technol ogies
```
Comment (muzixing) [inline draft-ietf-cats-framework.md:81]: Technologies

**Review by muzixing — COMMENTED** [review]
I will prefer to make the Abbreviations shorter, CSCI-ID seens too long and hard in pronounce. CATS SCI-ID? or SCI-ID ?

File: draft-ietf-cats-framework.md (hunk 1 above)
Comment (muzixing) [inline draft-ietf-cats-framework.md:81]:
```suggestion
  org: Huawei Technologies
```

**Comment by muzixing** [comment]
let's merge this, and ask for comments of the CSIC-ID.

**Pull request merged**